### ISSUE TITLE
Allow growing of MessageViewController

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/GitHawk#1613]
![GitHawk Upload by BasThomas](https://i.imgur.com/LaGmDXk.jpg)

I don’t know if this is a MessageViewController issue or just set to very low in GitHawk? Can we make it more flexible so we can grow it on iPad (especially with a hardware keyboard)?

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.18.0 (1519351495)
Device: iPad7,3 (iOS 11.2.6)
TestFlight: true
</pre>
</details>

**Comment by rnystrom** [comment]
Def! There’s a setting for the max height. Could set that larger if iPad. Maybe the default behavior is a little more advanced (like based on available screen space?).

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Yeah, that'd be neat. I would think maybe amount of lines or just a height. The consumer can then calculate whatever height it wants (eg. 30% of available space). MessageViewController can then be responsible to check if a keyboard is up or not?

**Comment by KashishGoel** [comment]
What about dynamically resizing the message field as you type up to a certain height? Should be able to a check to see if the height needs expanding using sizeToFit and then updating a constraint.
